src/cljs/nnts2/note/api.cljs

```diff
@@ -1,15 +1,19 @@
 (ns nnts2.note.api
-  (:require [ajax.core :refer [GET POST]]
+  (:require [ajax.core :as ajax]
             [re-frame.core :as re-frame]))
 
 (enable-console-print!)
 
-(defn create-note [data]
-  (POST "/note/create"
-        {:response-format :json
-         :keywords? true
-         :params (first data)
-         :format :json
-         :handler #(do (re-frame/dispatch [:note-submit-success])
-                       (js/alert "note created successfully"))
-}))
+(defn create-note-req-map [data]
```

Comment:
Renaming the namespace to something like `api-effects` would allow you to refer to these functions like `api-effects/create-note`. You can then drop the awkward `-req-map` suffix.

Reply:
This is done, have created a namespace api-data for anytime we are constructing an api call